Gopkg.toml

```diff
@@ -40,14 +45,14 @@
 
 [[constraint]]
   name = "github.com/go-redis/redis"
-  version = "6.13.2"
+  version = "6.15.1"
 
 [[constraint]]
   branch = "master"
   name = "github.com/korovkin/limiter"
 
 [[constraint]]
-  branch = "master"
+  branch = "move-lang-code"
```

Comment:
you cant merge with this change

Reply:
This is in continuation of the discussion around testing with new protocols. We won't merge it this way

Reply:
changed to "development"